docs/ecosystem/changelog.md

```diff
@@ -0,0 +1,22 @@
+---
+id: changelog
+title: Changelogs
+---
+
+> Running security-critical open source technology in a self-service format requires vigilance. We make your life easier by
+> announcing important software updates via the [Ory Security Newsletter](http://eepurl.com/di390P). Never miss an update and
+> **[sign up now to important release updates!](http://eepurl.com/di390P)**
+
+Ory is actively being developed. If you are interested in future development, take a look at the [Roadmap](./roadmap.md). Below
+you find changelogs for Ory services:
+
+- [Ory Kratos Releases](https://github.com/ory/kratos/releases) /
+  [Full CHANGELOG.md](https://github.com/ory/kratos/blob/master/CHANGELOG.md)
+- [Ory Hydra Releases](https://github.com/ory/hydra/releases) /
+  [Full CHANGELOG.md](https://github.com/ory/hydra/blob/master/CHANGELOG.md)
+- [Ory Keto Releases](https://github.com/ory/keto/releases) /
+  [Full CHANGELOG.md](https://github.com/ory/keto/blob/master/CHANGELOG.md)
+- [Ory Oathkeeper Releases](https://github.com/ory/oathkeeper/releases) /
+  [Full CHANGELOG.md](https://github.com/ory/oathkeeper/blob/master/CHANGELOG.md)
+
+Before [upgrading](./upgrading.md) to a newer version, please make sure to check with these documents first.
```

Comment:
How about you put it in an admonition and place it BEFORE the changelogs? Could look something like this: 

```
:::warning

Changelogs contain important information about the changes and breaking changes introduced in the code. Please read the relevant project's changelog before you [upgrade](./upgrading.md) to a new version.

:::